pkg/api/postage.go

```diff
@@ -287,6 +287,7 @@ func (s *Service) postageGetStampBucketsHandler(w http.ResponseWriter, r *http.R
 		}
 		return
 	}
+	defer func() { _ = save() }()
```

Comment:
I would recommend at least logging the error so that there is a crumb if it fails to save.